data_acquisition/src/general_functions.R

```diff
@@ -1,9 +1,9 @@
 library(yaml)
```

Comment:
I'm wondering if these library calls are redundant with the environment set up by `_targets.R`?

Reply:
Oh, that would be cool! I just assumed that you had to set up in `_targets.R` AND then call the library! I'll edit!